TransferList/TransferList.js

```diff
@@ -140,6 +140,15 @@ const TransferListBase = kind({
 		 */
 		noMultipleDrag: PropTypes.bool,
 
+		/**
+		 * The orientation for the transfer list.
+		 *
+		 * @type {Component}
```

Comment:
```suggestion
		 * @type {('horizontal', 'vertical')}
```

see Button example:
![image](https://user-images.githubusercontent.com/63335823/216059417-b7617048-2781-4def-8f8e-b7d689584a51.png)